packages/react-ui-core/src/Gmap/utils/const.js

```diff
@@ -0,0 +1,25 @@
+export const EVENTS = {
```

Comment:
Did you mean to remove this from somewhere else? Also, the name `const.js` or `EVENTS` doesn't make sense here. What are these events in reference to? The map? The markers? All of them? Thats what I would assume.